### DISCUSSION TITLE
How to add icon search to my website?

### DISCUSSION THREAD

**Discussion by html5syt** [squidfunk/mkdocs-material#7094] [Q&A] [answered]
Because of China's GFW, I can't visit github.io in many time, so it's very difficult for me to search for icons in official documents.

So I want to add it to myself docs, and I can use it anyway (I add custom domain in my repository).
How to do that?
![Screenshot_20240424_222628_mark via](https://github.com/squidfunk/mkdocs-material/assets/53525043/0a86097a-2e4c-4fca-9e36-3bb310d22a70)

**Comment by alexvoss** [comment, marked as answer]
You could simply clone the [mkdocs-material](https://github.com/squidfunk/mkdocs-material) repository and run `mkdocs serve` for a locally hosted copy of the documentation. Note that this will not be an exact copy of the hosted documentation but the emoji search will be there. Added bonus: all the other info.
